test/suites/dev/test-fees/test-fee-multiplier-xcm.ts

```diff
@@ -178,7 +178,7 @@ describeSuite({
           .as_v2();
 
         await injectHrmpMessageAndSeal(context, 1, {
-          type: "XcmVersionedXcm",
+          type: "StagingXcmVersionedXcm",
```

Comment:
I'm curious, where did this `"Staging"` part come from? I ask because I've had issues with polkadot+XCM types...

Reply:
I guess it is due to the changes in the names of `xcm`, `xcm-builder` and `xcm-executor`. They all now have a `staging-` prefix.